.github/workflows/deploy-infra.yaml

```diff
@@ -357,10 +363,20 @@ jobs:
           helm repo add fluxcd https://charts.fluxcd.io
           kubectl apply -f https://raw.githubusercontent.com/fluxcd/helm-operator/master/deploy/crds.yaml
 
-          [[ $ENVIRONMENT_NAME = "Biomage" ]] && FLUX_GIT_READ_ONLY=false || FLUX_GIT_READ_ONLY=true
+          FLUX_GIT_READ_ONLY=false
+          GIT_URL=git@github.com:${{ env.RELEASES_REPOSITORY }}
+
+          if [[ $ENVIRONMENT_NAME = "HMS" ]]
```

Comment:
if we are going to do this for flare, then it's probably to have `if [[ $ENVIRONMENT_NAME != "Biomage" ]]` ?

Reply:
good point, I will flip the logic of the if statement

Reply:
done